official/recommendation/ncf_keras_benchmark.py

```diff
@@ -277,6 +278,16 @@ def benchmark_xla_1_gpu_ctl_mlperf_like(self):
     FLAGS.train_epochs = 7
     self._run_and_report_benchmark_mlperf_like()
 
+  def benchmark_xla_1_gpu_ctl_fp16_mlperf_like(self):
```

Comment:
also a benchmark without XLA but with fp16?

Reply:
sure, will add that I did not wanted to add them gradually. But I can add it now as well.